tests/test_cadquery.py

```diff
@@ -3622,6 +3636,42 @@ def testNormal(self):
 
         self.assertTupleAlmostEquals(n.toTuple(), (0, 0, 1), 6)
 
+        r = Workplane().rect(1, 2).wires().val()
+        n = r.normal()
+
+        self.assertTupleAlmostEquals(n.toTuple(), (0, 0, 1), 6)
+
         with self.assertRaises(ValueError):
             edge = Workplane().rect(1, 2).edges().val()
             n = edge.normal()
+
+    def testPositionAt(self):
+
+        w = Workplane().lineTo(0, 1).lineTo(1, 1).wire().val()
```

Comment:
I'm assuming it doesn't matter if this wire is closed, but thought I'd check to be sure.

Reply:
Good point, I extended the test with a closed wire case.